### ISSUE TITLE
How to restart supabase services when I changed config.toml

### ISSUE THREAD

**Issue by jingsam** [supabase/cli#2078]
**Link**
Add a link to the page which needs improvement (if relevant)

**Describe the problem**
When I enable analytics service in config.toml, how can I make supabase cli start service?
Maybe I can do with `supabase stop` and `supabase start`, but that would be recreate all service which is slow. Maybe an `restart` command like `docker restart` is quicker?

**Describe the improvement**
A clear and concise description of the improvement.

**Additional context**
Add any other context or screenshots that help clarify your question.

**Comment by sweatybridge** [comment]
Thanks for the suggestion. I think we can have `supabase start` compare the list of running services vs enabled services in `config.toml`. It can then launch newly enabled containers incrementally.

**Comment by jingsam** [comment]
> Thanks for the suggestion. I think we can have `supabase start` compare the list of running services vs enabled services in `config.toml`. It can then launch newly enabled containers incrementally.

Yes, just like k8s `kubetcl apply config.yml`, autodiff and action to keep local services status match the `config.toml`

**Comment by zhanziyang** [comment]
+1 for this

**Comment by Allaoua9** [comment]
Would be nice to have a way to make quick restarts.

For now i'm using makefile with command shortcuts:
`make restart`

```
start:
	npx supabase start

stop:
	npx supabase stop

restart:
	npx supabase stop
	npx supabase start
```
